src/views/Hearing/index.js

```diff
@@ -128,12 +129,20 @@ class Hearing extends React.Component {
     const hearingAllowsComments = !hearing.closed && (new Date() < new Date(hearing.close_at));
     const onPostVote = this.onVoteComment.bind(this);
     const introSection = find(hearing.sections, (section) => section.type === "introduction");
-    const closureInfoSection = find(hearing.sections, (section) => section.type === "closure-info");
+    let closureInfoSection = find(hearing.sections, (section) => section.type === "closure-info");
```

Comment:
Would probably be better to let this remain `const` and move the rvalue and the fallback logic (below) into a helper function like `_getClosureInfoSection()`.